### DISCUSSION TITLE
Not all voices I have installed seem to be available in the  Text to speech section of preferences after installing Mac OS Ventura?

### DISCUSSION THREAD

**Discussion by va3ets200** [BearWare/TeamTalk5#1557] [Q&A]
Previously, under Mac OS Monterey, I was using Alex as my TTS voice in TTS preferences in TeamTalk.  Since upgrading to Mac OS Ventura, that does not seem to be available, and voices listed under American English voices, are actually Australian voices.  I had to redownload Alex to get it to be available so I could use it with VoiceOVer, but it doesn't seem to be  showing up in Teamtalk's preferences?  Very strange, not sure why this would be.

**Comment by bear101** [comment]
When I go into "VoiceOver Utility" on Ventura I don't see any voice called "Alex".

**Comment by va3ets200** [comment]
You have to go into  manage voices, and download it actually.

**Comment by va3ets200** [comment]
But in general, Teamtalk is saying that voices I know I have installed, cannot be found for some reason

**Comment by bear101** [comment]
You can try searching for the issue here: https://bugreports.qt.io/browse

**Comment by va3ets200** [comment]
And I have lots of American English voices installed.  And when I select American English as the  language in teamTalk preferences, the only voices that show up, are three Australian voices.  So something is definitely not right here, ever since upgrading to Ventura

**Comment by va3ets200** [comment]
Attachment available until Dec 16, 2022
Ok.  Will try that.  As well, here’s a recording, demonstrating what is happening.  Click to Download
 <https://www.icloud.com/attachment/?u=https%3A%2F%2Fcvws.icloud-content.com%2FB%2FAY1OvLAw4tnmZ-BJ8gmVuCOS9nyyAYsalabBGygLH6Z8sDOSuYx17lcT%2F%24%7Bf%7D%3Fo%3DAv-qOXzpQKClOWJRrYBYIlqpsek3MiKOgD1yHSG6WRP7%26v%3D1%26x%3D3%26a%3DCAogEO4CM2iD2wxnu4RP0v_SnqTfsvyJqY5paYQnfvm1c-ASeBCTktSQyDAYk6LP5NEwIgEAKgkC6AMA_wYCvRtSBJL2fLJaBHXuVxNqJot9o3iNClqI0HS6b5h_sMWkRU_lzmigivXo0cT2zEoXfODQAEJJciYeKK53ms-m4MuDSiufigR-Hoa6ggjSmR3HgkMDK2ku9HJQ7Afk2A%26e%3D1671221727%26fl%3D%26r%3D917A7C77-F1A5-4C0A-9B87-6511425397E9-1%26k%3D%24%7Buk%7D%26ckc%3Dcom.apple.largeattachment%26ckz%3D39CCA07B-BD27-4188-A683-07DC8866E937%26p%3D102%26s%3D1LH59DOIpvN7latjmkk8e3yStEs&uk=dKJUbeK0i0fAHQDn0TtWBA&f=What%20is%20going%20on%20with%20TTS%20section%20of%20TeamTalk%20preferences%20in%20Mac%20OS%20Ventura.m4a&sz=61919865>What is going on with TTS section of TeamTalk preferences in Mac OS Ventura.m4a
61.9 MB

**Comment by va3ets200** [comment]
In searching for this issue on that page you gave me a link to, it doesn’t appear that there are any issues relating to this on there.
